test/test_cuda.py

```diff
@@ -3417,6 +3417,7 @@ def test_matmul_device_mismatch(self):
                     torch.addmm(s, m1, m2)
 
     @unittest.skipIf(TEST_MULTIGPU, "Testing on one GPU is sufficient")
+    @skipIfRocm(msg="Skipped!!, since it is failing on ROCm 6.0")
```

Comment:
This sounds like a regression which will affect distributed workflows pretty badly, but up to you guys